src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -102,6 +107,12 @@ function WorkspaceInvitePage(props) {
         _.each(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
 
         const newSelectedOptions = [];
+        _.each(_.keys(props.invitedEmailsToAccountIDsDraft), (login) => {
+            if (!_.has(detailsMap, login)) {
+                return;
+            }
+            newSelectedOptions.push({...detailsMap[login], isSelected: true});
+        });
```

Comment:
This logic caused minor regression.
More details about the root cause: https://github.com/Expensify/App/issues/36350#issuecomment-1942208390